public/index.html

```diff
@@ -13,13 +13,20 @@
         <div class="form-field">
           <h3>Welcome to SimpleChat</h3>
         </div>
+        <div class="form-field">
+          <label>Gender</label>
+          <span>
+            <input type="radio" name="avatar" value="male.png" checked style="margin-left: 25px;" /> Male
```

Comment:
In the future these radio buttons will be hidden, but they will control the avatars in the same way.
Instead of radio buttons I will display actual pictures and the selected one will be colorful, while others will be black and white.